$NetBSD$

Add compile and linker flags for NetBSD.
Fix shell script = tests.

--- configure.ac.orig	2017-07-02 00:40:29.000000000 +0000
+++ configure.ac
@@ -392,6 +392,9 @@ AS_IF([test "$have_sdl2" = "yes"], [
      *-linux*)
        SDL_CFLAGS="$SDL_CFLAGS -D_GNU_SOURCE=1"
        ;;
+     *-netbsd*)
+       SDL_CFLAGS="$SDL_CFLAGS -D_GNU_SOURCE=1"
+       ;;
    esac
 # Disable mkspr
    SDL_IMAGE_CFLAGS=
@@ -412,6 +415,9 @@ case "$host" in
   *-linux*)
     GLOBAL_LIBS="$NAEV_LIBS -lm" # feels so wrong
     ;;
+  *-netbsd*)
+    GLOBAL_LIBS="$NAEV_LIBS -lm"
+    ;;
   *-mingw*)
     GLOBAL_LIBS="$NAEV_LIBS -lwsock32"
     ;;
@@ -470,10 +476,10 @@ esac
 # automake to detect when the binary needs recompiling after a library has been
 # changed.
 NAEV_DEPENDENCIES="$WIDGET_LIBS"
-AS_IF([test "$used_lua" == "internal"], [
+AS_IF([test "$used_lua" = "internal"], [
    NAEV_DEPENDENCIES="$NAEV_DEPENDENCIES $LUA_LIBS"
 ])
-AS_IF([test "$used_csparse" == "internal"], [
+AS_IF([test "$used_csparse" = "internal"], [
    NAEV_DEPENDENCIES="$NAEV_DEPENDENCIES $CSPARSE_LIBS"
 ])
 
